src/NServiceBus.AcceptanceTesting/AcceptanceTestingTransport/AcceptanceTestingTransport.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NServiceBus
+{
+    using Routing;
+    using Settings;
+    using Transport;
+
+    public class AcceptanceTestingTransport : TransportDefinition, IMessageDrivenSubscriptionTransport
```

Comment:
What is the impact of adding `IMessageDrivenSubscriptionTransport` and then turning native pubsub on? I know it means the `RegisterPublisher` API will be available when it's not needed. Will calling it when using native pubsub cause a problem, or will the entries just be ignored?

Is there anything else having this interface does?

Reply:
that interface is only used to give access to the publisher registration API, it doesn't have any further impact.